Add bloom-filter with length (#43)
apache/parquet-testing: data/README.md

| delta_encoding_optional_column.parquet       | optional INT64 and STRING columns with delta encoding. See [delta_encoding_optional_column.md](delta_encoding_optional_column.md) for details.                   |
| nested_structs.rust.parquet                  | Used to test that the Rust Arrow reader can lookup the correct field from a nested struct. See [ARROW-11452](https://issues.apache.org/jira/browse/ARROW-11452)  |
| data_index_bloom_encoding_stats.parquet | optional STRING column. Contains optional metadata: bloom filters, column index, offset index and encoding stats.                                                |

> | data_index_bloom_encoding_with_length.parquet | Same with data_index_bloom_encoding_stats.parquet but has bloom_filter_length populated in the ColumnMetaData |

Reviewer: ```suggestion
| data_index_bloom_encoding_with_length.parquet | Same as `data_index_bloom_encoding_stats.parquet` but has `bloom_filter_length` populated in the ColumnMetaData |
```

> | data_index_bloom_encoding_with_length.parquet | optional STRING column. Contains optional metadata: bloom filters with bloom_filter_length, column index, offset index and encoding stats.                                                |

Reviewer 2: ```suggestion
| data_index_bloom_encoding_with_length.parquet | Same with data_index_bloom_encoding_stats.parquet but has bloom_filter_length populated in the ColumnMetaData |
```
What about this?
